.gitignore

```diff
@@ -72,6 +72,9 @@ test/cilium-[0-9a-f]*.yaml
 test/*tmp
 test/cilium-istioctl
 
+# Testdata, includes generated .o
+!**/testdata/*
```

Comment:
Is there no way we can avoid committing these binary files?

Reply:
Good question! Stuffing binary files in Git is generally frowned upon, but:
- These files are generated by a single command using a stable, Dockerized toolchain
- CI always clean-builds the files to ensure there's no diff
- They're stripped as far down as possible, leaving just the ELF structure, BTF blob and the binary contents of the prog and map sections. The objects in this repo are max 4000 bytes large. For reference, this repo's vendor/ is currently a whopping 162 MiB of source code.

This buys us lightning-fast `go test` runs without any runtime toolchain or other external dependencies, a trade-off that's well worth it.